libs/elements/layout/footer/src/lib/components/footer/footer.component.scss

```diff
@@ -0,0 +1,78 @@
+.footer
+{
+    background-color: rgb(48, 48, 85);
+    padding: 5%;
+}
+
+.footer__links
+{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    margin-top: 10px;
+    border-top: 1px solid #aaaaaa;
```

Comment:
Use color variables from the variables.scss file. 
If they are not provided for, add them there.